app/repositories/CommonBSONDocuments.scala

```diff
@@ -106,6 +106,7 @@ trait CommonBSONDocuments extends BaseBSONReader {
           fastPassAccepted = getProgress(ProgressStatuses.FAST_PASS_ACCEPTED.key),
           withdrawn = getProgress(ProgressStatuses.WITHDRAWN.key),
           sdipFSFailed = getProgress(FailedSdipFsTestType.progressStatus),
+          sdipFSSuccessful = getProgress(SuccessfulSdipFsTestType.progressStatus),
```

Comment:
ProgressStatus has an implicit toString method defined so the .key calls are redundant.  I'll update the SDIP calls to make them the same though